.travis.yml

```diff
@@ -73,14 +74,12 @@ deploy:
   on:
     tags: true
     repo: kubevirt/kubevirt
-- provider: pages
+- provider: script
   skip_cleanup: true
-  repo: kubevirt-incubator/api-reference
-  target_branch: master
-  github_token: $API_REFERENCE_PUSH_TOKEN
-  local_dir: hack/gen-swagger-doc/html5
+  script: hack/gen-swagger-doc/deploy.sh
```

Comment:
maybe it needs to be executable or prefix it with `bash` command ?

Reply:
Let's try with bash prefix

Reply:
ok I will update it